content/en/docs/concepts/services-networking/service.md

```diff
@@ -824,6 +824,13 @@ If you want to map a Service directly to a specific IP address, consider using
 [headless Services](#headless-services).
 {{< /note >}}
 
+{{< note >}}
+If the external service has a valid domain name and you don’t need port remapping, 
+then using the `ExternalName` service type is an easy and quick way to map the external 
+service to an internal one. If you don’t have a domain name or need to do port remapping, 
+add the IP addresses to an `EndpointSlice` and use that instead.
+{{< /note >}}
+
```

Comment:
```suggestion
{{< note >}}
If the external service has a valid domain name and you don’t need port remapping, 
then using the `ExternalName` service type is convenient to map the external 
service to an internal one. Otherwise,
add the IP addresses to an `EndpointSlice` and use that instead.
{{< /note >}}

```

Reply:
small nits @swastik959

Reply:
@Ritikaa96 done